test/type/builtin_types.swift

```diff
@@ -0,0 +1,4 @@
+// RUN: %target-typecheck-verify-swift
+
+let x = 0
+_ = x._value == x // expected-error {{referencing operator function '==' on 'BinaryInteger' requires that 'Builtin.Int64' conform to 'BinaryInteger'}}
```

Comment:
So, the comment about tests in the Jira was about not relying on the implementation details of Int (the `_value` property). Instead, you can make some other type in this file that uses Builtin.Int64 (or whatever) as a stored property by adding `-parse-stdlib` to the test invocation. (You'll then need to `import Swift` to still get access to the normal standard library.)

Reply:
No problem